lib/facter/puppet_environmentpath.rb

```diff
@@ -0,0 +1,21 @@
+# This facter fact returns the value of the Puppet environmentpath. It is adapted
+# from the puppetlabs-stdlib puppet_vardir fact.
```

Comment:
In https://github.com/theforeman/puppet-puppet/blob/fc1d1b3b5310d6caaaf6a654c4875aa327bfe94d/templates/server/post-receive.erb#L13 we get this from the configuration but that may be easier since that module also configures puppet itself.